pkg/gardenlet/controller/shoot/care/health.go

```diff
@@ -511,6 +516,15 @@ func (h *Health) checkObservabilityComponents(
 		return exitCondition, nil
 	}
 
+	filterFunc := func(managedResource resourcesv1alpha1.ManagedResource) bool {
+		return managedResource.Labels[v1beta1constants.LabelCareConditionType] == string(gardencorev1beta1.ShootObservabilityComponentsHealthy) &&
+			managedResource.Labels[managedresources.LabelKeyManagedBy] == managedresources.LabelValueGardenlet
+	}
```

Comment:
Why do we need this? We should only care about the Prometheis in the shoot namespace here anyways?